Message Board/View Controllers/MessageThreadsTableViewController.swift

```diff
@@ -46,6 +48,7 @@ class MessageThreadsTableViewController: UITableViewController {
         let cell = tableView.dequeueReusableCell(withIdentifier: "MessageThreadCell", for: indexPath)
         
         cell.textLabel?.text = messageThreadController.messageThreads[indexPath.row].title
+        cell.accessibilityLabel = "MessageThreadsTableVC.Cell\(indexPath)"
```

Comment:
This is a great way to add extra hints, but use the `accessibilityIdentifier`, since the label is user facing with VoiceOver